src/components/Dashboard/FilingHistoryList/CompletedDissolution.vue

```diff
@@ -31,17 +32,23 @@ export default class CompletedDissolution extends Mixins(DateMixin) {
 
   @Getter isCoop!: boolean
   @Getter getEntityName!: string
+  @Getter getDissolutionConfirmationResource!: DissolutionConfirmationResourceIF
 
   /** The entity title to display. */
   get entityTitle (): string {
-    return this.isCoop ? 'Cooperative Association' : 'Company'
+    return this.getDissolutionConfirmationResource?.entityTitle
```

Comment:
Since this may evaluate as undefined, I recommend a fallback value (eg, Company or Business -- check with UI/UX).

Reply:
Implemented a switch case for the different business types.

Reply:
The value is coming from the business-config file so I believe it is always defined.

Reply:
OK. Please remove the conditional chaining (`?`) as that makes it look like the resource could be null or undefined.